filesmanager/filesmanager.py

```diff
@@ -789,6 +789,39 @@ def delete_asset(self, asset_key):
         except AssetNotFoundException as e:  # pylint: disable=broad-except
             log.exception(e)
 
+    @XBlock.json_handler
+    def download_content(self, data, suffix=''):
```

Comment:
Remember adding docstrings to methods & modules

Reply:
corrected